src/main/java/br/com/sbk/sbking/gui/elements/AllDirectionBoardElements.java

```diff
@@ -28,6 +28,8 @@ public AllDirectionBoardElements(Deal deal, Container container, ActionListener
         new LeaveTableElement(container, new Point(150, 50), actionListener);
 
         new SpectatorsElement(container, FrameConstants.spectatorNamesPosition, spectators);
+
+        new GameNameElement(gameName, container, new Point(container.getWidth() / 2, 10), actionListener);
```

Comment:
I am pretty sure that the screen half width is defined in the FrameConstants. If not, you may create that to avoid repeating this calculation in all places.